Process Process-2:
Traceback (most recent call last):
  File "/home/jmcates/miniconda3/envs/zmcint/lib/python3.6/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/home/jmcates/miniconda3/envs/zmcint/lib/python3.6/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/home/jmcates/zmc-numba/ZMCIntegral.py", line 114, in multi_processing
    result.append(self.MCkernel(domain, i_batch))
  File "/home/jmcates/zmc-numba/ZMCIntegral.py", line 376, in MCkernel
    integration_kernel[blockspergrid, threadsperblock](num_loops,                                                           MCresult,                                                           chunk_size,                                                           n_chunk_x,                                                           domain,                                                           domain_range,                                                           batch_size,                                                           i_batch,                                                           rng_states,                                                           n_chunk)
  File "/home/jmcates/miniconda3/envs/zmcint/lib/python3.6/site-packages/numba/cuda/compiler.py", line 767, in __call__
    cfg(*args)
  File "/home/jmcates/miniconda3/envs/zmcint/lib/python3.6/site-packages/numba/cuda/compiler.py", line 497, in __call__
    sharedmem=self.sharedmem)
  File "/home/jmcates/miniconda3/envs/zmcint/lib/python3.6/site-packages/numba/cuda/compiler.py", line 608, in _kernel_call
    wb()
  File "/home/jmcates/miniconda3/envs/zmcint/lib/python3.6/site-packages/numba/cuda/args.py", line 65, in <lambda>
    retr.append(lambda: devary.copy_to_host(self.value, stream=stream))
  File "/home/jmcates/miniconda3/envs/zmcint/lib/python3.6/site-packages/numba/cuda/cudadrv/devices.py", line 212, in _require_cuda_context
    return fn(*args, **kws)
  File "/home/jmcates/miniconda3/envs/zmcint/lib/python3.6/site-packages/numba/cuda/cudadrv/devicearray.py", line 235, in copy_to_host
    _driver.device_to_host(hostary, self, self.alloc_size, stream=stream)
  File "/home/jmcates/miniconda3/envs/zmcint/lib/python3.6/site-packages/numba/cuda/cudadrv/driver.py", line 1853, in device_to_host
    fn(host_pointer(dst), device_pointer(src), size, *varargs)
  File "/home/jmcates/miniconda3/envs/zmcint/lib/python3.6/site-packages/numba/cuda/cudadrv/driver.py", line 290, in safe_cuda_api_call
    self._check_error(fname, retcode)
  File "/home/jmcates/miniconda3/envs/zmcint/lib/python3.6/site-packages/numba/cuda/cudadrv/driver.py", line 325, in _check_error
    raise CudaAPIError(retcode, msg)
numba.cuda.cudadrv.driver.CudaAPIError: [719] Call to cuMemcpyDtoH results in CUDA_ERROR_LAUNCH_FAILED
Traceback (most recent call last):
  File "./linalg-zmc-test2.py", line 55, in <module>
    result = MC.evaluate()
  File "/home/jmcates/zmc-numba/ZMCIntegral.py", line 75, in evaluate
    MCresult = self.importance_sampling_iteration(self.initial_domain, 0)
  File "/home/jmcates/zmc-numba/ZMCIntegral.py", line 81, in importance_sampling_iteration
    MCresult_chunks, large_std_chunk_id, MCresult_std_chunks = self.MCevaluate(domain)
  File "/home/jmcates/zmc-numba/ZMCIntegral.py", line 133, in MCevaluate
    MCresult.append(np.load(os.getcwd()+'/multi_temp/result'+str(i_batch)+'.npy'))
  File "/home/jmcates/miniconda3/envs/zmcint/lib/python3.6/site-packages/numpy/lib/npyio.py", line 422, in load
    fid = open(os_fspath(file), "rb")
FileNotFoundError: [Errno 2] No such file or directory: '/home/jmcates/zmc-numba/multi_temp/result0.npy'
srun: error: gpu-0-0: task 0: Exited with exit code 1
